apps/web/pages/booking/[uid].tsx

```diff
@@ -545,6 +547,14 @@ export default function Success(props: SuccessProps) {
                         </div>
                       </>
                     )}
+                    {bookingInfo?.responses && (
```

Comment:
I created a meeting where "What is this meeting about?" is hidden and booked the meeting, and the "What is this meeting about?" still appears:

<img width="736" alt="Screenshot 2023-12-14 at 09 48 22" src="https://github.com/calcom/cal.com/assets/42170848/edc8737d-ea5e-4b54-b989-0e13294c63e4">

So we need the following check:
```
bookingInfo?.responses?.title && typeof bookingInfo?.responses?.title === "string"
```

First, we check if the title is defined.
Next, we check if it's a string so that we do not have to typecast it as `any`.

Reply:
Changes added :)